### ISSUE TITLE
G.722WB codec support

### ISSUE THREAD

**Issue by cintema** [LubosD/twinkle#99]
Unlinke in Linphone, the G.722WB codec (which has become somewhat become a de facto standard in the recent years) appears to still not being supported by twinkle.
Would it be feasible to incorporate G.722 support in the next twinkle release?
Thank you!
Twinkle is a great client (unlke Linphone :-))

**Comment by sinyb** [comment]
Just a +1 for this.

**Comment by fbriere** [comment]
I have just submitted #177 which adds G.722 support.  Unfortunately, I was barely able to test it myself, so if anybody could give it a try, that would be most welcome!  (The PR itself contains more details about where I might have screwed up.)

**Comment by sjlongland** [comment]
I just did a test here… between this branch of Twinkle on a Gentoo Linux/AMD64 laptop and a Grandstream GXP1615 IP phone via Asterisk.  It worked perfectly.

I had Twinkle using the JACK ALSA plug-in… and was piping audio from Clementine through to the input of Twinkle and hearing that out the desk phone.  Later this week I plan to do similar, but Clementine will be swapped with a VoIP chat client (Cisco WebEX if I can get it working, or Jitsi), as my radio club plans to use a VoIP group-chat service for meetings (thanks to COVID-19, we can't meet in person) and one of our members does not have the Internet at home.

I did try a combined G.722+Opus branch as well, but that hung after the call was established, so it would appear G.722 and Opus do not get along at this time.  Maybe that can be worked-on later (or maybe I cocked up the merge).
